src/xdocs/config_whitespace.xml

```diff
@@ -1244,7 +1244,7 @@ public void foo(final char @NotNull [] param) {} // No violation
           <tr>
             <td>allowLineBreaks</td>
             <td>
-              whether whitespace is allowed if the token is at a linebreak
+              Control whether whitespace is flagged at linebreaks.
```

Comment:
> allowed if the token is at a linebreak
> flagged at linebreaks

I am on the fence about which wording to use.
Property is named `allow` so the word `flagged` seems like the negative form of the property when reading.
Its like saying "disallowed at linebreaks` when we name the property `allow`.

Reply:
agree, existing description is more clear as for me.

Reply:
@pbludov please use the old description

Reply:
done